tests/devices/qubit/test_apply_operation.py

```diff
@@ -480,23 +473,41 @@ def test_parametrized_evolution_state_vector_return_intermediate(self, mocker):
         assert spy.call_count == 2
         assert qml.math.allclose(state_ev, state_rx, atol=1e-6)
 
-    def test_batched_state_raises_an_error(self):
-        """Test that if is_state_batche=True, an error is raised"""
+    @pytest.mark.parametrize("num_state_wires", [2, 4])
```

Comment:
This parametrization is in order to hit both, the matrix and the state evolution branch.